src/example-types/esTwoLevelAggregation.js

```diff
@@ -30,19 +36,17 @@ module.exports = {
               context.key_data
             )
           ),
-          aggs: {
-            twoLevelAgg: {
-              [context.value_type]: _.omitBy(
-                _.isNil,
```

Comment:
Are we sure we don't need this anymore?

Reply:
I will put it back in but I'm not 100% sure is even there.